modules/sdk/src/main/scala/org/tessellation/sdk/domain/snapshot/services/GlobalL0Service.scala

```diff
@@ -111,6 +111,10 @@ object GlobalL0Service {
         type Result = LatestSnapshotTuple
         for {
           peers <- globalL0ClusterStorage.getPeers.map(nes => Random.shuffle(nes.toSortedSet))
+
+          _ <- logger.debug(s"Pulling latest snapshot using ${peers.size} peers")
+          _ <- logger.trace(s"${peers.show}")
```

Comment:
Even if it's a trace log I would still log a short peer id format if it's not the case right now.

Reply:
There is a very little chance that there will be a clash on e.g. first 5 chars for Id. Hence logging more than that seems redundant.